docs/swagger.yaml

```diff
@@ -292,7 +341,7 @@ paths:
       required: true
       type: "string"
       description: "The agreement identifier (id)"
-  /api/v1/charge/{agreementId}/{chargeId}:
+  /api/v1/charges/{agreementId}/{chargeId}:
```

Comment:
/api/v1/agreements/{agreementId}/charges/{chargeId} ?

Reply:
How will you change all these URL and be compatible with Infosoft/Fuel of Norway.

Reply:
If they are using the autogenerated swagger client APIs, they should just need to regenerate them :) Hopefully its part of their build pipeline and not code thats checked in to their repository

Reply:
I don’t think anyone does that unless it’s been part of technical/API specifications and possibly in agreements as well. Would love to be pleasantly surprised though.